exchanges/wshandler/websocket.go

```diff
@@ -13,9 +13,9 @@ import (
 	log "github.com/thrasher-/gocryptotrader/logger"
 )
 
-// WebsocketInit initialises the websocket struct
-func (e *Base) WebsocketInit() {
-	e.Websocket = &Websocket{
+// Init initialises the websocket struct
+func Init() *Websocket {
```

Comment:
This is a really minor thing but what are your thoughts on remaing this to New()?

init() in go is used for package level initialization and having Init() can cause confusion most people tend to go with New (and we use this same naming convetion in other places such as the requestor package)

Again minor thing that has no impact on execution and happy with either way

Reply:
Good spotting. Yeah I would rename.

Reply:
Done!

Reply:
Awesome :D

With that change the exchange template file will need to be updated as well

https://github.com/gloriousCode/gocryptotrader/blob/around-the-world/tools/exchange_template/main_file.tmpl#L49

After that everything else seems great and good to go!

Reply:
Another good catch! I've updated it along with some other minor changes to that template file.